doc/tutorials/Configure_EDR_service.md

```diff
@@ -23,14 +23,14 @@ This file is available in the adaguc-server repository with location `data/datas
 Create the following file at the filepath `$ADAGUC_DATASET_DIR/edr.xml`. You can also consider changing `<FilePath>` to `/data/adaguc-data/*.nc`.
 
 ```xml
-<?xml version="1.0" encoding="UTF-8" ?>
+
 <Configuration>
 
     <OgcApiFeatures/>
 
     <OgcApiEdr>
         <EdrCollection name="harmonie">
-            <EdrParameter name="air_temperature__at_2m" unit="Celsius"/>
+            <EdrParameter name="air_temperature__at_2m" unit="°C"  standard_name="air_temperature" label="Air temperature, 2 metre"/>
```

Comment:
If we are matching up a layer anyway, why not use the `units` from the layer variable. I guess the (?) reason is that the layer variable `units` is optional in this config file, and we don't want to do a call into adaguc C++ when getting back `/collections`?

Reply:
It is indeed optional. So we cannot use that. Unless we start reading the GetCapabilities document.......

Reply:
This information is not avaiable in the GetCapabilities document. Created an issue https://github.com/KNMI/adaguc-server/issues/374